maintainers/maintainer-list.nix

```diff
@@ -5022,6 +5022,11 @@
     github = "fkautz";
     githubId = 135706;
   };
+  FlafyDev = {
```

Comment:
Can you add `name = "Flafy Arazi";`?

Seems as though the CI fails because there is no `name`.